tests/e2e/test_e2e.py

```diff
@@ -240,19 +239,44 @@ def test_prompt_manager_no_model_params():
         )
 
 
-def test_prompt_manager_no_model_params_undeployed():
+def test_prompt_manager_undeployed_latest_revision():
     mgr = UsedByCiDontDeleteNoParamsUndeployedPromptManager(
-        UsedByCiDontDeleteNoParamsUndeployedMinorVersion.DANGEROUSLY_USE_UNDEPLOYED,
+        # Need to use a user-scoped API key to access undeployed prompts
+        api_key=os.environ["AUTOBLOCKS_API_KEY_USER"],
+        # Request the latest revision
+        minor_version="latest",
     )
 
     with mgr.exec() as prompt:
-        with pytest.raises(AttributeError):
-            # This should raise an AttributeError because the prompt has no params
-            prompt.params.model  # type: ignore
-
         assert prompt.track()["id"] == "used-by-ci-dont-delete-no-params"
-        assert prompt.track()["version"] == "undeployed"
-        assert prompt.track().get("params") is None
+        assert prompt.track()["version"].startswith("revision:")
+
+
+def test_prompt_manager_undeployed_specific_revision():
+    """
+    This test uses a revision created in our CI org:
+
+    https://app.autoblocks.ai/prompts/used-by-ci-dont-delete-no-params/revisions/clvifur81000336hksycpw4av/edit
+    """
+    mgr = UsedByCiDontDeleteNoParamsUndeployedPromptManager(
+        # Need to use a user-scoped API key to access undeployed prompts
+        api_key=os.environ["AUTOBLOCKS_API_KEY_USER"],
```

Comment:
end-users won't need to have a separate env var with the _USER suffix, i just have it set up this way for our e2e tests so we can test both an org-scoped and user-scoped API key

end-users will just have an `AUTOBLOCKS_API_KEY` env var that is set to their org-scoped API key in prod and to user-scoped API keys in local dev